ios/Podfile.lock

```diff
@@ -203,6 +203,8 @@ PODS:
     - React-jsi (= 0.64.1)
```

Comment:
Let's remove the all .lock files in both iOS and Android.

Reply:
Done!